src/edu/washington/escience/myria/operator/DbDelete.java

```diff
@@ -0,0 +1,64 @@
+/**
+ *
+ */
+package edu.washington.escience.myria.operator;
+
+import com.google.common.collect.ImmutableMap;
+
+import edu.washington.escience.myria.DbException;
+import edu.washington.escience.myria.MyriaConstants;
+import edu.washington.escience.myria.RelationKey;
+import edu.washington.escience.myria.accessmethod.AccessMethod;
+import edu.washington.escience.myria.accessmethod.ConnectionInfo;
+import edu.washington.escience.myria.storage.TupleBatch;
+
+/**
+ * 
+ * TODO: it's not really an "operator", is it?
```

Comment:
Certainly making `DELETE` an "operator" is an awkward fit, but more importantly Myria's distributed query plans aren't transactional--see below.